content/hacking-atom/sections/writing-specs.md

```diff
@@ -19,37 +19,43 @@ Spec files **must** end with `-spec` so add `sample-spec.coffee` to the `spec` d
 
 The `describe` method takes two arguments, a description and a function. If the description explains a behavior it typically begins with `when`; if it is more like a unit test it begins with the method name.
 
-```coffee
-describe "when a test is written", ->
-  # contents
+```javascript
+describe("when a test is written", function() {
```

Comment:
Personally at least I would say to use arrow functions in the specs as well, since the specs already have a lot of verbosity to the code. Leaving it as is is fine though.

Reply:
Thanks for all the help! I'm probably going to leave the tests as they are but I'll think about it 👍